_posts/2020-12-01-configuring-and-suppressing-warnings.md

```diff
@@ -0,0 +1,192 @@
+---
+layout: blog-detail
+post-type: blog
+by: Lukas Rytz
+title: "Configuring and suppressing warnings in Scala"
+isHighlight: true
+---
+
+Scala 2.13.2 and 2.12.13 introduced the `-Wconf` compiler flag to globally configure reporting of warnings, and the `@nowarn` annotation to locally suppress them. Having more control over compiler warnings makes them a lot more valuable:
+
+* In projects where the build log shows a lot of warnings that are mostly ignored, new helpful warnings can easily go undetected. The new functionality can be used to clean up the build log with manageable efforts and potentially enable fatal warnings (`-Werror` in 2.13, `-Xfatal-warnings` in 2.12). This has happened for example in the Scala compiler and standard library projects in the past few months – thank you, [@NthPortal](https://github.com/NthPortal)!
```

Comment:
maybe thank Som too? not sure what level of effort he's put into this compared to Nth's

Reply:
I didn't follow these PRs in detail, so I'm sorry for having missed @som-snytt's contribution here.